Kajabi/sage-lib
# Adding development dependencies for deployments.

## Description
The SassDoc and Storybook deploys for Sage were broken due to missing development dependencies. I added several development dependencies to `sage-react` and added one to `sage-assets`. Preliminary testing showed a successful deploy, but we won't know for sure if it worked until after the next version bump.

Added back the steps to deploy Storybook/SassDocs in the circleci manifest.

## Testing

- Checkout the branch and verify the local development environment still works as expected.
- Verify the Kajabi Products bridge still works as expected

## Merging

We're merging this directly into master as it doesn't need to be buffered with other changes.
